tools/js-sdk-release-tools/src/llc/utils/generateChangelog.ts

```diff
@@ -71,15 +71,15 @@ export async function generateChangelog(packagePath) {
                 const newSDKType = getSDKType(packagePath);
                 const changelog = await extractExportAndGenerateChangelog(apiMdFileNPM, apiMdFileLocal, oldSDKType, newSDKType);
                 if (!changelog.hasBreakingChange && !changelog.hasFeature) {
-                    logger.error('Cannot generate changelog because the codes of local and npm may be the same.');
+                    logger.error('Failed to generate changelog because the codes of local and npm may be the same.');
                 } else {
                     appendChangelog(packagePath, version, changelog);
-                    logger.info('Generate changelog successfully');
+                    logger.info('Generated changelog successfully.');
                 }
             }
 
         } catch (e: any) {
-          logger.error(`Generate changelog failed: ${e.message}`);
+          logger.error(`Failed to generate changelog: ${e.message}.`);
```

Comment:
What are the next steps when changelog fails to be generated? Or can this error be ignored?

Reply:
it will be replace by `generateChangelogAndBumpVersion` function in `automaticGenerateChangeLogAndBumpVersion.ts` in pipeline, you can ignore it.